extensions/wikia/PortableInfoboxBuilder/PortableInfoboxBuilderHooks.class.php

```diff
@@ -65,7 +65,8 @@ public static function onMakeGlobalVariablesScript( array &$aVars ) {
 
 		if ( $wgEnableVisualEditorExt && \VisualEditorHooks::isAvailable( \RequestContext::getMain()->getSkin() ) ) {
 			$aVars['wgEnablePortableInfoboxBuilderInVE'] = $wgEnablePortableInfoboxBuilderInVE &&
-				\RequestContext::getMain()->getUser()->isAllowed( 'createpage' );
+				// use dummy title in template ns
+				Title::newFromText( 'ATemplate', NS_TEMPLATE )->userCan( 'createpage' );
```

Comment:
The whole idea is to check if the user can create any template, this is the easiest way of doing that and it "should" return the correct value. I haven't thought about the case when `ATemplate` is protected to be edited only by admins and moderators. Do you have any proposal on what title could we use? On the other hand, I would recommend to a static name as it would be easier to debug in case of bugs.